cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -258,6 +258,19 @@ def __setitem__(self, view: ViewIdentifier, properties: MutableMapping[PropertyI
         view_id = ViewId.load(view)
         self.data[view_id] = properties
 
+    def _repr_html_(self) -> str:
+        pd = local_import("pandas")
+        index_names = "space", "view", "version"
```

Comment:
For nodes/edges, there will never be multiple versions existing simultaneously as version is just an integer that increments every time something in the node/edge changes. Thus, maybe not worth to include in an index?

Reply:
This is the version of the view

Reply:
My mistake. Then I agree it makes sense. 

I guess the alternative would be to just make one index of `space:externa_id/version` but I do not mind the multiindex.